spec/features/social_features_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+feature "Social networking" do
+    given(:jane) { Fabricate(:user) }
+    given(:joe) { Fabricate(:user) }
+    given(:tv_comedies) { Fabricate(:category, name: 'TV Comedies') }
+
+  scenario "following a user" do 
+    futurama = Fabricate(:video, title: 'Futurama', category: tv_comedies)
+    jane_review = Fabricate(:review, video: futurama, user: jane)
+    
+    sign_in(joe)
+    expect(page).to have_content "Welcome, #{joe.full_name}"
```

Comment:
This expectation isn't actually germane to the feature test. The danger here is that you are setting an expectation that you may change in the future (and might even test for elsewhere). There are valid use cases for this type of test in that if ajax is involved you could use this to check if the page is loaded or it may fail because it didn't wait. I still don't suggest asserting on the actual text though. I would suggest a check on a css value over plain text like this.